### DISCUSSION TITLE
PiPedal v1.1.21 arm64

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#94] [Releases]
- Substantial improvements to CPU use and under-runs for TooB Convolution Reverb and TooB Cab IR

Bug fixes:
- TooB CabIR loads wrong file for Impulse file 2 and 3.
- Wifi Direct setup fails.
- Wrong file types for upload dialog on Android web and Android PiPedal client.
-

**Comment by 38github** [comment]
Glad to see updates! I had to reconfigure the http port after I installed this one over 1.1.20. Is it possible to make updates care about this setting?

**Reply by rerdavies** [reply]
Where did you reconfigure the port? In pipedal.service? Using the pipedalconfig uitlity? Or in a settings file somewhere (I know there is a setting for it, I just can't remember where it is off the top of my head).

**Reply by rerdavies** [reply]
ps. I logged a bug to that effect. Expect it to be addressed in the next release (although I suspect you might have to customize it one more time).

**Reply by 38github** [reply]
I used pipedalconfig.
